src/reference/00-Getting-Started/03-Directories.md

```diff
@@ -61,7 +61,7 @@ form the complete build definition. See [organizing the build][Organizing-Build]
 ```
 build.sbt
 project/
-  Build.scala
+  Dependencies.scala
```

Comment:
I'd vote for not talking about the `project/Foo.scala` format until later in the `Organizing-Build` section. After all, it's purely an organizational/neatness thing, and isn't necessary to learn so early